cluster-provision/k8s/1.25/k8s_provision.sh

```diff
@@ -131,6 +131,30 @@ rules:
   - Panic
 EOF
 
+#
+cat > /etc/kubernetes/psa.yaml <<EOF
```

Comment:
need to be done for cluster-provision/k8s/1.25-ipv6/k8s_provision.sh
as well please